tests/behat/fixtures/civictheme_migrate.node_civictheme_page_3.json

```diff
@@ -0,0 +1,278 @@
+[
+  {
+    "id": "77d61781-0657-4f9a-9d12-173ea03eae91",
+    "title": "[TEST] Migrated Page Content 1",
+    "alias": "/migrated/page-content-1",
+    "summary": "<span>Designed by government, for government. Our goal is to help you deliver and maintain secure, modern and user-friendly websites. GovCMS is community-based, but don't just take our word for it.</span>",
+    "topics": "[TEST] Topic 1,[TEST] Topic 2,[TEST] Topic 3,[TEST] Topic 4",
+    "thumbnail": ["c97a9b08-f3b0-477b-97f7-8b61f9d4a527"],
+    "vertical_spacing": "top",
+    "hide_sidebar": true,
+    "show_last_updated_date": true,
+    "last_updated_date": "1 July 2020",
+    "banner": {
+      "type": "container",
+      "children": [
+        {
+          "theme": "dark",
+          "title": "[TEST] Banner title - Migrated Page Content 1",
+          "blend_mode": "darken",
+          "featured_image": ["7fdce6fd-3bcb-4ffa-b349-2a6eb0b049c4"],
+          "background": ["7966cf0a-598d-4da2-8443-837fccd46109"],
+          "hide_breadcrumb": true
+        }
+      ]
+    },
+    "content": {
+      "type": "container",
+      "children": [
+        {
+          "text_content": {
+            "format": "rich_text",
+            "value": "<p>[Content migration] - Product and Services Guide</p>"
+          },
+          "manual_list": {
+            "type": "manual_list",
+            "children": {
+              "cards": {
+                "type": "container",
+                "children": [
+                  {
+                    "promo_card": {
+                      "item_title": "[TEST] Promo card 1",
+                      "item_content": {
+                        "format": "rich_text",
+                        "value": "<p>[TEST] Promo card summary</p>"
+                      }
+                    },
+                    "event_card": {
+                      "item_title": "[TEST] Event card 1",
+                      "item_content": {
+                        "format": "rich_text",
+                        "value": "<p>[TEST] Event card summary</p>"
+                      },
+                      "item_topics": "[TEST] Event Topic 1,[TEST] Topic 2"
+                    },
+                    "publication_card": {
+                      "item_title": "[TEST] Publication card 1",
+                      "item_content": {
+                        "format": "rich_text",
+                        "value": "<p>[TEST] Publication card summary</p>"
+                      },
+                      "item_document": [
+                        "448f7b0e-19a3-4c43-b1d7-5b1f196dbf98"
+                      ]
+                    },
+                    "service_card": {
+                      "item_title": "[TEST] Service card 1",
+                      "item_links": [
+                        {
+                          "uri": "http://example.com",
+                          "title": "[TEST] Service card link 1"
```

Comment:
`[TEST] Service card link 1` should be `[TEST] Service card 1 link 1`

so that below you could do `[TEST] Service card 2 link 1`, `[TEST] Service card 2 link 2` and then assert in test to make sure that all links are coming through